docs/cli-client/bank/burn.md

```diff
@@ -2,7 +2,7 @@
 
 ## Description
 
-Burn tokens 
+Decrease the balance of input address, so that the fund couldn't be recovered
 
```

Comment:
This command is used for burning tokens from specified address. Anyone can send this transaction type. There is no any connection to fund.

Reply:
fix